src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,7 @@
+html {
+  height: 100%;
+}
```

Comment:
```suggestion
```

Reply:
If I do it, the clock will not be aligned vertically.

Reply:
in this case better to use `100vh`